pkg/miniogw/config.go

```diff
@@ -112,9 +112,9 @@ func (c Config) action(ctx context.Context, cliCtx *cli.Context,
 	return Error.New("unexpected minio exit")
 }
 
-// NewGateway creates a new minio Gateway
-func (c Config) NewGateway(ctx context.Context,
-	identity *provider.FullIdentity) (gw minio.Gateway, err error) {
+// GetBucketStore returns an implementation of buckets.Store
+func (c Config) GetBucketStore(ctx context.Context,
+	identity *provider.FullIdentity) (bs buckets.Store, err error) {
```

Comment:
Avoid breaking lines in unnatural places. https://github.com/golang/go/wiki/CodeReviewComments#line-length